src/db/user-suspension.ts

```diff
@@ -0,0 +1,53 @@
+import { Moderator, moderatorSchema } from './moderator';
+import { Schema, Document } from 'mongoose';
+import { User } from 'discord.js';
+import { ObjectId } from 'mongodb';
+import { Collections } from './collections';
+
+/**
+ * A user-suspension is used to keep track of suspensions
+ * made on a user by admins.
+ */
+export interface UserSuspension extends Document {
+  /**
+   * The parent user
+   *
+   * **NOTE** I'm unsure if we should keep this as the objectId reference
+   * or use the discord provided snowflake
+   */
+  user: string | ObjectId | User;
+  /**
+   * The reason the user was suspended.
+   */
```

Comment:
Again, I think we need to determine what value will fill this field so we can be more strict with the typing.